src/Loggers/LogImage.jl

```diff
@@ -1,14 +1,42 @@
 """
     log_image(logger, name, img, step)
-
-Logs an image with name `name` at step `step`
+Logs an image
 """
-function log_image(logger::Logger, name::String, img; step=nothing)
-    summ    = SummaryCollection()
+function log_image(logger::TBLogger, name::AbstractString, img::Array; step=nothing)
+    summ = SummaryCollection()
     push!(summ.value, image_summary(name, img))
     write_event(logger.file, make_event(logger, summ, step=step))
 end
 
-function image_summary(name::String, img)
-    Summary_Image(height = size(img)[1], width = size(img)[2], colorspace = 3, encoded_image_string = Nothing)
+function image_summary(name::AbstractString, img::Array)
+    #Encode image to string
+    io = IOBuffer()
+    iob64_encode = Base64.Base64EncodePipe(io)
+    write(iob64_encode, img)
+    close(iob64_encode)
+    str = String(take!(io));
+    eis = Vector{UInt8}(str)
+    #obtain height and width of image
+    height = size(img)[1]
+    width = size(img)[2]
```

Comment:
```
height = size(img)[1]
width = size(img)[2]
```

Reply:
This is an old code. Please check out latest commit

Reply:
My bad. I'm on a train with shitty internet.